### PR TITLE
PyPi package, updated results and pep8 for everything 

### PR DESCRIPTION
**Pull request by danpol**
1. Trained all models on the full dataset
2. Updated metrics
3. Added a script to print tables (csv / latex / html)
4. pep8 for everything
5. added a shiny badge for Travis CI :) 
6. created PyPi package `molsets`: just run `pip install molsets`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +165,9 @@ For comparison of molecular properties, we computed the Frèchet distance betwee
 
 # Installation
 
+### PyPi
+The simplest way to install MOSES (models and metrics) is to install [RDKit](https://www.rdkit.org/docs/Install.html): `conda install -yq -c rdkit rdkit` and then install MOSES (`molsets`) from pip (`pip install molsets`).
```
Comment (shayakhmetov) [inline README.md]: rdkit только через conda install возможно не установится правильно, с `apt-get install libxrender1 libxext6` будет достаточно

File: moses/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.1.1'
+__version__ = '0.1.0'
```
Comment (shayakhmetov) [inline moses/__init__.py]: тут точно 0.1.0?

Comment (danpol) [reply moses/__init__.py]: Да 🤷‍♂️

File: README.md (hunk 1 above)
Comment (danpol) [reply README.md]: Travis works without these libraries... Probably RDKit uses them only for drawing.

File: scripts/run.py [hunk 3]
```diff
@@ -73,14 +73,15 @@ def get_parser():
...
+    print('Training...')
     model_path = get_model_path(config, model)
     config_path = get_config_path(config, model)
     vocab_path = get_vocab_path(config, model)
 
-    if os.path.exists(model_path) and \
-            os.path.exists(config_path) and \
-            os.path.exists(vocab_path):
-        return
+#     if os.path.exists(model_path) and \
```
Comment (shayakhmetov) [inline scripts/run.py]: Why as a comment?

File: README.md (hunk 1 above)
Comment (shayakhmetov) [reply README.md]: Maybe, travis's image already has these packages. In my experiments, RDKit didn't use them only for drawing

**Pull request merged**